### PR TITLE
Make manuscript-ready figures

### PR DESCRIPTION
**Pull request by ajlee21**
This PR uses svg utils (inspired by @danich1  [annorxiver repo](https://github.com/greenelab/annorxiver/tree/master/figure_generation)) to make manuscript figures.

Many files were touched, but the only one to review is `figure_generation/figure_generation.ipynb` and output figures in `figure_generation/output/*`

As a note, please let me know what you think about the color schemes...i'm not exactly sold on it.

### CODE REVIEW

**Comment by vincerubinetti** [comment]
@ajlee21 can you attach a zip file of all the figures you want me to review so I know exactly which ones to look at (and also so i dont have to download this pr; i wish github showed the svgs in the web interface)

**Comment by ajlee21** [comment]
@vincerubinetti do these links work?

[Figure 1](https://github.com/greenelab/generic-expression-patterns/blob/1ba7d79bdf5ed0334eeb85339643d23d80274bc6/figure_generation/output/figure_1.svg)
[Figure 2](https://github.com/greenelab/generic-expression-patterns/blob/1ba7d79bdf5ed0334eeb85339643d23d80274bc6/figure_generation/output/figure_2.svg)
[Figure 3](https://github.com/greenelab/generic-expression-patterns/blob/1ba7d79bdf5ed0334eeb85339643d23d80274bc6/figure_generation/output/figure_3.svg)

**Comment by vincerubinetti** [comment]
Ah they do, and I didn't realize I could just click this button in the diff to see the files, sorry:

image

**Comment by vincerubinetti** [comment]
The figures look good to me. Anything I came up with is nitpicks and isn't strictly necessary.

image

image

image

**Comment by vincerubinetti** [comment]
Good colors to use:
https://www.materialpalette.com/colors

**Comment by vincerubinetti** [comment]
Also for future reference, i think graphs look nicer and are easier to read when they have their edges go from center to center, like this:

image

**Review by vincerubinetti — APPROVED** [review]

File: environment.yml [hunk 1]
```diff
@@ -33,3 +33,4 @@ dependencies:
   - matplotlib==3.0.0
   - ponyo==0.2
   - matplotlib-venn
+  - svgutils
```
Comment (danich1) [inline environment.yml:36]: Been good to specify the version for svgutils and matplotlib-venn so you dont run into issues.

File: figure_generation/nbconverted/figure_generation.py [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+# Create panels for figure 1
+panel_1a = make_figure_panel(
+    "fig1A.svg",
+    2.5, 
+    2.5,
+    30,
+    10
+)
+panel_1b = make_figure_panel(
+    "../human_general_analysis/logs/NN_2500_30/tybalt_2layer_30latent_hist.svg",
```
Comment (danich1) [inline figure_generation/nbconverted/figure_generation.py:69]: Would suggest integrating Pathlib, but not necessary for this PR.

**Review by danich1 — APPROVED** [review]
Just add versions to packages. Great use of svgutils!!

**Comment by ajlee21** [comment]
> Good colors to use:
> https://www.materialpalette.com/colors

Thank you for taking a look. I'll try to make some of the changes you suggested. I actually used this link to pick these colors:) But I'm not sure if I like the overall palette I chose. I'll have to think about it some more.

I also agree that the workflow diagrams are pretty squished, but this is because I didn't have enough room on google slides.

File: figure_generation/nbconverted/figure_generation.py (hunk 2 above)
Comment (ajlee21) [reply figure_generation/nbconverted/figure_generation.py:69]: Very much agree. I really like Pathlib -- a much cleaner way to set parameters. Something for the next project!

**Comment by danich1** [comment]
This might also be a good link to try if you are looking for color palettes: https://colorbrewer2.org/
This is how I picked the color scheme for my paper.

**Pull request merged**